quarkus/config-api/src/main/java/org/keycloak/config/SecurityOptions.java

```diff
@@ -18,6 +19,17 @@ public class SecurityOptions {
             .defaultValue(FipsMode.DISABLED)
             .build();
 
+    public static final Option<String> TRUSTSTORE_PATHS = new OptionBuilder<>("truststore-paths", String.class)
+            .category(OptionCategory.SECURITY)
+            .description("List of pkcs12 (.p12 or .pfx), PEM files, or directories containing those files that will be used as a global truststore.")
+            .build();
+
+    public static final Option<HostnameVerificationPolicy> HOSTNAME_VERIFICATION_POLICY = new OptionBuilder<>("hostname-verification-policy", HostnameVerificationPolicy.class)
+            .category(OptionCategory.SECURITY)
+            .description("The hostname verification policy.")
```

Comment:
We should probably elaborate in the desc a bit more. Especially reflect it's for outgoing TLS.

Reply:
Updated to mention out-going HTTPS and SMTP.